.github/workflows/nix-jupyterlab.yml

```diff
@@ -21,6 +21,6 @@ jobs:
       with:
         name: hasktorch
         signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
-        skipNixBuild: false
-        file: nix/jupyter-shell.nix
-        nixBuildArgs: --option sandbox false 
+    - run: |
+        cachix use iohk
+        nix-build nix/jupyter-shell.nix -j 2 --option sandbox false 
```

Comment:
cachix-action@v6 does not support nixBuildArgs. 
cachix-action@v6  uploads caches in post action.